<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavbarTest</class>
 <widget class="QWidget" name="NavbarTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Navbar" name="total" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Navbar" name="onlyTitle" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Navbar" name="noBack" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Navbar" name="noFunc" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Navbar" name="userBar" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 127);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Navbar</class>
   <extends>QWidget</extends>
   <header location="global">navbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
